packages/nimble-components/src/wafer-map/tests/computations.spec.ts

```diff
@@ -0,0 +1,234 @@
+import { Computations } from '../modules/computations.module';
+import { Margin, WaferMapQuadrant } from '../types';
+import { getWaferMapDies } from './utilities';
+
+describe('Computations module', () => {
+    let computationsModule: Computations;
+
+    describe('with 100 square canvas', () => {
+        const defaultMargin: Margin = {
+            top: 20,
+            right: 20,
+            bottom: 20,
+            left: 20
+        };
+        beforeEach(async () => {
```

Comment:
It doesn't look like any of the beforeEach blocks in the file require `async` behavior, so we can just remove the `async` modifier from this spot and the others below

```suggestion
        beforeEach(() => {
```